INVENTORY.md

```diff
@@ -0,0 +1,22 @@
+# Inventory
+
+this page talks about common inventory information information provided by DPU vendors in a same format
+
+- Questions:
```

Comment:
What is the destinction between inventory and discovery?  Discovery seems to imply remote discovery.

There also seems to be a preceeding question of connectivity and which elements are required.